package.json

```diff
@@ -36,6 +36,8 @@
     "start:docker:widget": "cross-env nx run-many --target=start:docker --projects=@novu/widget",
     "docker:build": "pnpm -r --if-present --parallel docker:build",
     "build": "nx run-many --target=build --all",
+    "rebuild": "npm run prebuild && npm run build",
```

Comment:
There is a 'pnpm rebuild' reserved command, however, it did not serve the purpose of the 'rebuild' (delete old build and then build again) therefore I created these duplicated commands across the repositories.

In addition due the reserved command  'pnpm rebuild' in order to run this script, you need to run 'npm run rebuild'